target/linux/ramips/dts/mt7620a_trendnet_tha103ac.dts

```diff
@@ -0,0 +1,199 @@
+#include "mt7620a.dtsi"
```

Comment:
please add the SPDX header top of the dts file

Reply:
Done, thanks.